python/builtInExample2.py

```diff
@@ -0,0 +1,144 @@
+    [1.3744, 0.000242441, 0.0, 1.51061, -0.00983316, 0.0, 1.65137, -0.0434185, 0.0, 1.70586, -0.181118, 0.0],
+    [1.21547, -0.701691, 0.0, 1.27904, -0.614369, 0.0, 1.51061, -0.607652, 0.0, 1.52878, -1.16853, 0.0, 1.53786, -0.688257, 0.0],
+    [-0.310174, -0.325535, 0.0, -0.0150337, -0.124023, 0.0, 0.216538, -0.140816, 0.0, 0.466272, -0.399423, 0.0]
+]
+
+# Save the four range space control polygons
+polygonTubes = []
+
+# Save the corresponding four fiber surfaces
+fiberSurfaces = []
+
+for coords in rangePolygonsCoordinates:
+    # create a new 'Poly Line Source'
+    polyLineSource1 = PolyLineSource()
+    polyLineSource1.Points = coords
+    
+    # create a new 'Resample With Dataset'
+    resampleWithDataset1 = ResampleWithDataset(SourceDataArrays=tTKContinuousScatterPlot1,
+        DestinationMesh=polyLineSource1)
+    resampleWithDataset1.CellLocator = 'Static Cell Locator'
+
+    # create a new 'Tetrahedralize'
+    tetrahedralize2 = Tetrahedralize(Input=resampleWithDataset1)
+
+    # create a new 'TTK FiberSurface'
+    tTKFiberSurface1 = TTKFiberSurface(InputDomain=tetrahedralize1,
+        RangePolygon=tetrahedralize2)
+    tTKFiberSurface1.DomainUComponent = ['POINTS', 'log(Rho)']
+    tTKFiberSurface1.DomainVComponent = ['POINTS', 'log(s)']
+    tTKFiberSurface1.PolygonUComponent = ['POINTS', 'log(Rho)']
+    tTKFiberSurface1.PolygonVComponent = ['POINTS', 'log(s)']
+    tTKFiberSurface1.WithPointMerging = 1
+
+    # create a new 'Generate Surface Normals'
+    generateSurfaceNormals1 = GenerateSurfaceNormals(Input=tTKFiberSurface1)
+    
+    fiberSurfaces.append(generateSurfaceNormals1)
+
+    # create a new 'Extract Surface'
+    extractSurface1 = ExtractSurface(Input=resampleWithDataset1)
+
+    # create a new 'Tube'
+    tube3 = Tube(Input=extractSurface1)
+    tube3.Radius = 0.03
+    
+    polygonTubes.append(tube3)
+
+# compute 'TTK JacobiSet' for the bivariate data
+tTKJacobiSet1 = TTKJacobiSet(Input=tetrahedralize1)
+tTKJacobiSet1.UComponent = ['POINTS', 'log(Rho)']
+tTKJacobiSet1.VComponent = ['POINTS', 'log(s)']
+tTKJacobiSet1.Withedgeidentifiers = 1
+tTKJacobiSet1.Withvertexscalars = 1
+
+# project the Jacobi set onto the range space using 'TTK ProjectionFromField'
+tTKProjectionFromField3 = TTKProjectionFromField(Input=tTKJacobiSet1)
+tTKProjectionFromField3.UComponent = ['POINTS', 'log(Rho)']
+tTKProjectionFromField3.VComponent = ['POINTS', 'log(s)']
+
+# save the output
+SaveData('logRhoIsosurfaceAtoms.vtp', contour2)
+SaveData('logSIsosurfaceBonds.vtp', contour3)
+
+SaveData('ContinuousScatterPlot.vtu', threshold1)
+SaveData('logRhoIsosurfaceRangeProjection.vtp', tube1)
+SaveData('logSIsosurfaceRangeProjection.vtp', tube2)
+SaveData('OxygenAtomsRangePolygon.vtp', polygonTubes[0])
+SaveData('CarbonAtomsRangePolygon.vtp', polygonTubes[1])
+SaveData('CovalentBondsRangePolygon.vtp', polygonTubes[2])
+SaveData('NonCovalentIntercationSiteRangePolygon.vtp', polygonTubes[3])
```

Comment:
I would suggest to store the raw geometry instead of graphical primitives (tubes).